vendor/plugins/edu.harvard.dce.paella.heartbeatSender/heartbeat_sender.js

```diff
@@ -0,0 +1,68 @@
+// A version of the deprecated es.upv.paella.UserTrackingCollectorPlugIn, 
+// shaved down to just send the heartbeat.
+
+var classDef = {
```

Comment:
This is a very general variable name to define this specific plugin and doesn't match the style of the other dce plugins.